scripts/run-tests

```diff
@@ -11,7 +11,11 @@ cd $basedir/bin
 
 # Scan objects first so that does not happen within a test
 echo -e "\033[0;36mBuilding OpenRCT2 repository indexes...\033[0m"
-./openrct2 scan-objects
+if [[ $(uname) == "Darwin" ]]; then
+    ./openrct2-cli scan-objects
```

Comment:
I think this can be used in all cases

Reply:
I was thinking the same thing, but wasn't sure if openrct2-cli was built for all CI jobs. I'll try switching it for all and see what happens.

Reply:
@janisozaur looks like we're good - I've removed this conditional and always use openrct2-cli. All CI jobs just passed.